client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -106,14 +111,19 @@ export default function PlanOnlyThankYou( {
 						paragraph: <p />,
 						br: <br />,
 						strong: <strong />,
-						// eslint-disable-next-line jsx-a11y/anchor-is-valid
-						a: <a href="" onClick={ resendEmail } />,
+						a: <Button plain onClick={ resendEmail } />,
```

Comment:
I found out about the `plain` button variant by checking out the code for `@automattic/components/button`: https://www.npmjs.com/package/@automattic/components?activeTab=code